lib/assessment.md

```diff
@@ -95,7 +124,10 @@ Order the following parts of the box model from 'innermost' to 'outermost'.
 * padding
 
 <!-- your answer starts here -->
-
+margin
+border
+padding
+content
```

Comment:
The instructions said  
> "innermost to outermost"
but it's clear from the ordering that you got this one.